tests/phpunit/tests/l10n/wpTranslationsConvert.php

```diff
@@ -0,0 +1,598 @@
+<?php
+
+/**
+ * @coversDefaultClass WP_Translation_Controller
+ * @group l10n
+ * @group i18n
+ */
+class WP_Translation_Controller_Convert_Tests extends WP_UnitTestCase {
+	/**
+	 * @covers ::instance
+	 *
+	 * @return void
+	 */
+	public function test_get_instance() {
```

Comment:
All tests should have the `@ticket` annotation.

🔢 applies to all tests

Reply:
We typically only add these for bugfixes, not to _any_ test